### PR TITLE
Update README, fix import pkg

### PR DESCRIPTION
**Pull request by apiraino**
Hi,

I've just forked the repo because I was curious :-)

Please find in this pr a small fix on an import pkg. I've also updated the installation README info based on my usual Python workflow (which I find cleaner because it doesn't clutter your system with libraries you only need there).

Unfortunately I could not go futher than seeing the help screen because my Amiga skills are pretty rusty and every Amiga binary I tried to run failed with such error:

```
$ ./vamos 4D-Clock 
15:49:15.987       path:WARNING:  ami_to_sys_path: ami_path='4D-Clock' -> abs_path='SYS:4D-Clock' -> no resolved paths!
15:49:15.987       path:WARNING:  ami_to_sys_path: ami_path='c:4D-Clock' -> abs_path='c:4D-Clock' -> no resolved paths!
15:49:15.987       path:WARNING:  ami_command_to_sys_path: ami_path='4D-Clock' not found!
15:49:15.987       proc:  ERROR:  failed loading binary: failed mapping binary path: '4D-Clock'
```

Could you include a base tutorial guide on how to run a basic Amiga command?

Thanks a lot for your effort!

Antonio

### CODE REVIEW

File: amitools/vamos/lib/DosLibrary.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 from amitools.vamos.label.LabelStruct import LabelStruct
 from dos.CommandLine import CommandLine
 from amitools.vamos.Process import Process
-from dos.DosErrors import DosErrors
+from dos import Error
```
Comment (apiraino) [inline amitools/vamos/lib/DosLibrary.py:22]: small import fix

File: amitools/vamos/lib/DosLibrary.py [hunk 2]
```diff
@@ -30,8 +30,8 @@ class DosLibrary(AmigaLibrary):
 
   DOSFALSE = 0
   DOSTRUE = 0xffffffff
-  
-  LV_VAR   = 0	# an variable 
+
+  LV_VAR   = 0	# an variable
```
Comment (apiraino) [inline amitools/vamos/lib/DosLibrary.py:34]: All changes in this file were performed automatically by EMACS and are purely cosmetic.

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Cython==0.23.4
+py==1.4.31
+pytest==2.8.5
+sexpdata==0.0.3
+wsgiref==0.1.2
+
+# EMACS users, enable these too :)
+# epc==0.0.5
+# jedi==0.9.0
```
Comment (apiraino) [inline requirements.txt:9]: This file should simplify the library installation process.

**Comment by thorfdbg** [comment]
Hi Antonio,

to run the emulator, you need to create a .vamosrc file that specifies 
the configuration.

It defines the paths and volumes. Here's a shortened version of my 
configuration:

/\* snip */

[vamos]
ram_size=16384
stack_size=32
hw_access=disable
cpu=68020

[volumes]
SCSI=/store/Amiga
T=/tmp/amiga
ENV=/tmp/env

[assigns]
SYS=SCSI:
DEVS=SYS:Devs
L=SYS:L
S=SYS:S
C=SYS:C
LIBS=SYS:Libs

[path]

# every undefined assign "xy:" will be mapped to "system:xy/"

#auto_assign=system:
path=C:

# disable loading library in any case

[68040.library]
mode=off

# "auto" mode: either use amiga library if available and fall back

# to vamos internal lib if not amiga library was found

[icon.library]
mode=vamos

[utility.library]
mode=vamos

[locale.library]
mode=off

/\* snip */

You then run the shell as such:

vamos -x L:Shell-Seg

where the Shell-Seg is the Os 3.9 shell.

HTHH,
    Thomas

**Comment by apiraino** [comment]
thanks! Now that you mention that .rc file I could grep in your documentation and find the relevant parts (`doc/vamos.md` and `config/sample_vamosrc`).

**Comment by thorfdbg** [comment]
Am 31.12.2015 um 15:51 schrieb apiraino:

> Hi,
> 
> Could you include a base tutorial guide on how to run a basic Amiga command?
> 
> Thanks a lot for your effort!
> 
> The Amiga Shell first looks in the current directory, then in the path 
> (to be configured with PATH, or preferably in the .vamosrc). The path 
> also includes by default the C: directory, which is an assign. This also 
> has to be configured in the .vamosrc file.

Here is an example:

[volumes]
SCSI=/store/Amiga
T=/tmp/amiga/t
ENV=/tmp/env
RAM=/tmp/amiga

[assigns]
SYS=SCSI:
DEVS=SYS:Devs
L=SYS:L
S=SYS:S,V:S
C=SYS:C

[path]
path=.,C:

# disable loading library in any case

[68040.library]
mode=off

[icon.library]
mode=vamos

[utility.library]
mode=vamos

[locale.library]
mode=off

[intuition.library]
mode=vamos

This should give you the basic ability to run a command in C: or in the 
current directory.

BTW, you find a new version in github.

Greetings,
    Thomas

**Comment by apiraino** [comment]
Hi Thomas,
now the Shell executable is found and executed, although with errors:

```
$ ./vamos -x L:Shell-Seg
0.SYS:> dir
18:52:00.366       lock:WARNING:  lock 'dir' invalid: sys path does not exist: 'SYS:dir' -> '/store/Amiga/dir'
18:52:00.367       lock:WARNING:  lock 'dir' invalid: sys path does not exist: 'SYS:./dir' -> '/store/Amiga/./dir'
18:52:00.368       lock:WARNING:  lock 'dir' invalid: sys path does not exist: 'C:dir' -> '/store/Amiga/C/dir'
18:52:00.372       lock:WARNING:  lock 'dir' invalid: sys path does not exist: 'SYS:dir' -> '/store/Amiga/dir'
18:52:00.372       lock:WARNING:  lock 'dir' invalid: sys path does not exist: 'SYS:./dir' -> '/store/Amiga/./dir'
18:52:00.373       lock:WARNING:  lock 'dir' invalid: sys path does not exist: 'C:dir' -> '/store/Amiga/C/dir'
dir: Unknown command
```

but I think I need to populate the directories defined in the `[volumes]` directive with some commands.

I've also committed another minor fix of a typo I've encountered.

Thanks,
Antonio

**Pull request closed without merging**